libnetwork/netavark/testfiles/invalid/bad_metric.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "badmetric",
+    "id": "c73dab6e164c6cec6edad726406450f313b0a33fa59a59c4830a1f71793ef59f",
+    "driver": "bridge",
+    "network_interface": "podman101",
+    "created": "2021-10-06T18:50:54.25770461+02:00",
+    "subnets": [
+        {
+            "subnet": "10.89.100.0/24",
+            "gateway": "10.89.100.1",
+            "lease_range": {
+                "start_ip": "10.89.100.20",
+                "end_ip": "10.89.100.50"
+            }
```

Comment:
since you need to repush please remove this, it just adds unnecessary lines I as reviewer have to look at.